specification/maps/data-plane/Search/preview/2021-11-01-preview/geocoding.json

```diff
@@ -0,0 +1,686 @@
+    "ApiVersion": {
+      "name": "api-version",
+      "description": "Version number of Azure Maps API.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "default": "2021-11-01-preview",
+      "x-ms-parameter-location": "client"
+    },
+    "Top": {
+      "name": "top",
+      "in": "query",
+      "description": "Maximum number of responses that will be returned. Default: 5, minimum: 1 and maximum: 20.",
+      "required": false,
+      "type": "integer",
+      "default": 5,
+      "minimum": 1,
+      "maximum": 20,
+      "x-ms-parameter-location": "method"
+    },
+    "Accept-Language": {
+      "name": "Accept-Language",
+      "in": "header",
+      "description": "Language in which search results should be returned. \n\nPlease refer to [Supported Languages](https://docs.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/supported-culture-codes) for details.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "client"
+    },
+    "Location": {
+      "name": "location",
+      "in": "query",
+      "description": "A point on the earth specified as a latitude and longitude. When you specify this parameter, the user’s location is taken into account and the results returned may be more relevant to the user. Example: &location=lon,lat",
+      "required": false,
+      "type": "array",
+      "items": {
+        "type": "number",
+        "format": "double"
+      },
+      "minItems": 2,
+      "maxItems": 2,
+      "collectionFormat": "csv",
+      "x-ms-parameter-location": "method"
+    },
+    "Bbox": {
+      "name": "bbox",
+      "in": "query",
+      "description": "A rectangular area on the earth defined as a bounding box object. The sides of the rectangles are defined by latitude and longitude values. For more information, see Location and Area Types. When you specify this parameter, the geographical area is taken into account when computing the results of a location query.\n\nExample: lon1,lat1,lon2,lat2",
+      "required": false,
+      "type": "array",
+      "items": {
+        "type": "number",
+        "format": "double"
+      },
+      "collectionFormat": "csv",
+      "minItems": 4,
+      "maxItems": 4,
+      "x-ms-parameter-location": "method"
+    },
+    "View": {
+      "name": "view",
+      "in": "query",
+      "description": "A string that an [ISO 3166-1 Alpha-2 region/country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2). This will alter Geopolitical disputed borders and labels to align with the specified user region.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "CountryRegionSet": {
+      "name": "countryRegionSet",
+      "in": "query",
+      "description": "Comma separated string of country or region codes, e.g. FR,ES. This will limit the search to the specified countries. Only the first country code will be used for the Geocoder.",
```

Comment:
If only the first value will be used, why would the customer send a set of values?

Reply:
Also, are we following the ISO 3166-1 Alpha-2 standard here as well? If so, we should call it out.

Reply:
use multiple region here is for the future expansion. We discuss this when we define the api parameter.